meta-lmp-bsp/recipes-bsp/u-boot/u-boot-base-scr/imx8qm-mek/uEnv.txt.in

```diff
@@ -0,0 +1,4 @@
+bootcmd_dtb=fatload ${devtype} ${devnum}:1 ${fdt_addr_r} ${fdtfile}
```

Comment:
This one is broken, as I was testing lmp-base build manually typing commands, will fix it